src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -162,12 +162,15 @@ public function getPriority(): int
     public function isCandidate(Tokens $tokens): bool
     {
         return $tokens->isAnyTokenKindsFound([
-            T_FUNCTION,
-            T_DOC_COMMENT,
-            T_IMPLEMENTS,
-            T_EXTENDS,
             T_CATCH,
             T_DOUBLE_COLON,
+            T_DOC_COMMENT,
+            T_EXTENDS,
+            T_FUNCTION,
+            T_IMPLEMENTS,
+            T_INSTANCEOF,
+            T_NEW,
+            CT::T_USE_TRAIT,
```

Comment:
```suggestion
            CT::T_USE_TRAIT,
            T_CATCH,
            T_DOC_COMMENT,
            T_DOUBLE_COLON,
            T_EXTENDS,
            T_FUNCTION,
            T_IMPLEMENTS,
            T_INSTANCEOF,
            T_NEW,
```
I see you wanted to sort this list, so it here's some minor improvement. I suspect you wanted `CT::T_USE_TRAIT` to be at the end as you treated `CT::` part as a detail and looked only at `T_USE_TRAIT`, but IMHO it should be sorted naturally, with all custom tokens at the beginning.

Reply:
done